### PR TITLE
Bedsheet Wearing - Spooky - Click in hand to adjust, old interact been moved to alt click

### PR DESCRIPTION
**Pull request by JamieD1**
Bedsheet Wearing - Spooky - Click in hand to adjust, old interact been moved to alt click

#### Changelog

:cl:  
rscadd: Bedsheet Wearing - Spooky - Click in hand to adjust, old interact been moved to alt click
/:cl:
image

### CODE REVIEW

File: code/game/objects/structures/bedsheet_bin.dm [hunk 1]
```diff
@@ -20,6 +20,7 @@ LINEN BINS
 	w_class = WEIGHT_CLASS_TINY
 	item_color = "white"
 	resistance_flags = FLAMMABLE
+	var/suffic = null
```
Comment (jcatjmeow) [inline code/game/objects/structures/bedsheet_bin.dm]:
```suggestion
	var/suffix = null
```

**Review by jcatjmeow — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/bedsheet_bin.dm [hunk 2]
```diff
@@ -29,6 +30,22 @@ LINEN BINS
 		..()
 
 /obj/item/bedsheet/attack_self(mob/user)
+	if(bedsuffix)
+		var/newbedpath = "/obj/item/bedsheet/adjusted/" + bedsuffix
```
Comment (Amelia0010) [inline code/game/objects/structures/bedsheet_bin.dm]: There's no need for this to not just be a typepath

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/bedsheet_bin.dm (hunk 2 above)
Comment (JamieD1) [reply code/game/objects/structures/bedsheet_bin.dm]: > There's no need for this to not just be a typepath
Show me how great nicholas

**Review by Partheo — APPROVED** [review]
This looks kind of terrible honestly, lol. I'd be okay with it only remaining for Halloween or something.

**Comment by JamieD1** [comment]
> This looks kind of terrible honestly, lol. I'd be okay with it only remaining for Halloween or something.

CODER SPRITES GOOD

**Review by Amelia0010 — APPROVED** [review]

**Pull request merged**